lib/extension.js

```diff
@@ -56,78 +57,24 @@ class Extension {
      * @param {string} contents Contents of the template file to create
      * @param {string} descriptor Description of the template file (used in the prompt)
      * @param {string} file Filename
-     * @param {string} dir Directory to link the config file into
-     * @return {Promise<boolean>} True if the config file was created successfully, otherwise false
+     * @param {string} dir Directory to move the config file into
+     * @return {Promise} True if the config file was created successfully, otherwise false
```

Comment:
```suggestion
     * @return {Promise<void>}
```
Looks like it doesn't return a boolean anymore, so the description is no longer accurate 😅

Reply:
done 👍